source/_components/google_assistant.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Google Assistant"
+description: "Setup for Google Assistant integration"
+date: 2017-10-17 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google-assistant.png
+ha_category: Voice
+featured: true
+ha_release: 0.56
+---
+
+# Google Assistant Docs
+The Google Assistant component requires a bit more setup than most due to the way Google requires Assistant Apps to be set up.
+
+### {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+google_assistant:
+  project_id: someproject-2d0b8
+  client_id: [long url safe random string]
+  access_token: [a different long url safe random string]
+  exposed_domains:
+    - switch
+    - light
+    - group
+```
+
+*Note:* It's very important that you use very long strings for `client_id` and `access_token`. Those are essentially credentials to your Home Assistant instance.
+
+*Configuration Variables:*
+* *expose_by_default* (Optional): Expose devices in all supported domains by default.
+* *project_id* (Required): Project ID from the Google Developer console (looks like `words-2ab12`)
+* *client_id* (Required): A long random url safe string (no spaces or special characters) that will be used for Implicit OAuth.
```

Comment:
Can you also give an example here (like you do for `project_id`) for how this should look like?

Reply:
I get what you're saying but I was slightly skeptical about putting anything that looked like an example because I don't want anyone to use "password123" or "changethis" as their id or token.

Reply:
Sure, let us leave it out for now. If lot of people struggle with it, we can add it later.